File: src/main/java/eu/qwsome/sql/LeftTableJoin.java
package eu.qwsome.sql;

import eu.qwsome.sql.condition.Condition;

/**
 * Crate for left join attributes.
 *
 * @author Lukáš Kvídera
 */
class LeftTableJoin extends TableJoin {

  LeftTableJoin(final String joinTable, final Condition condition) {
    super(joinTable, condition);
  }

  @Override
  CharSequence getPrefix() {
    return " LEFT";
  }
}

File: src/main/kotlin/eu/qwsome/sql/LeftTableJoin.kt
/*
 * Crate for left join attributes.
 *
 * @author Lukáš Kvídera
 */
package eu.qwsome.sql

import eu.qwsome.sql.condition.Condition

internal class LeftTableJoin(
    joinTable: String,
    condition: Condition,
) : TableJoin(joinTable, condition) {

    override fun getPrefix(): CharSequence {
        return " LEFT"
    }
}
